app.yaml

```diff
@@ -1,6 +1,6 @@
 # dummy app.yaml for nosegae
 
-application: ndb-snippets
+application: python-docs-samples
```

Comment:
application and version field are supposed to be deprecated yes? Could we move these to env vars?